lib/marked.js

```diff
@@ -1082,6 +1082,14 @@ Parser.prototype.tok = function() {
     case 'text': {
       return this.renderer.paragraph(this.parseText());
     }
+    default: {
+      var errMsg = 'Token with "' + this.token.type + '" type was not found.';
```

Comment:
I think `this.token.type` can point to something different that what was used in the `switch` statement. Perhaps you should store it as a variable before calling `switch` to use in this message.

Also, what markdown input would cause `default` to be executed?

Reply:
Oh second thought, this looks correct.